rhc-ose-ansible/roles/openshift-management/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Validate OpenShift Token Provided
+  fail: msg="OpenShift Token Not Provided"
+  failed_when: (openshift_token is undefined) or (openshift_token is none) or (openshift_token|trim == '')
+  tags: always
+- name: Set Facts
+  set_fact: kubeconfig="/tmp/openshift-management-{{ ansible_date_time.epoch }}/config"
+  tags: always
+- name: Create Directory
+  file: path="{{ kubeconfig | dirname }}" state=directory
+  notify: cleanup openshift login
+  tags: always
+- name: Login to OpenShift
+  shell: oc login --token={{ openshift_token }} {{ openshift_login_insecure_flag }} {{ openshift_master_url }} 
```

Comment:
If we are using the token to login would it be better to not use login and just use the commands you normally would with the --token option? e.g. `oc adm prune deployments.... --token=<token>`